canonicalize.py

```diff
@@ -263,6 +263,10 @@ def sort_name_from_viaf(self, display_name, known_titles=None):
 
 class MockAuthorNameCanonicalizer(AuthorNameCanonicalizer):
 
+    """Mocks the services used by the author name canonicalizer, but
+    leaves the logic alone.
+    """
+
```

Comment:
We don't want to use this mock service when testing the controller because you still have to orchestrate the OCLC Linked Data and VIAF APIs. We want to mock the `canonicalize_author_name` method itself, so I made a new mock class.

I don't think this actually needs to be a mock class -- you should be able to pass mock OCLC Linked Data and VIAF APIs into the regular constructor.